appendix.tr.md

```diff
@@ -0,0 +1,39 @@
+
+[7]: https://web.archive.org/web/20210325014110/https://stallman.org/archives/2015-mar-jun.html#5_June_2015_(Law_being_an_ass)
+[8]: https://web.archive.org/web/20210325014131/https://stallman.org/archives/2014-jul-oct.html#26_October_2014_(Prison_for_cartoon)
+[9]: https://web.archive.org/web/20210325014249/https://stallman.org/archives/2012-nov-feb.html#04_January_2013_(Pedophilia)
+[10]: https://web.archive.org/web/20210325015259/https://stallman.org/archives/2019-jul-oct.html#14_September_2019_(Sex_between_an_adult_and_a_child_is_wrong)
+
+2015 ve 2016'da RMS, web sitesinde Down sendromu hakkında üç gönderi yaptı. Birisinin hamile olduğunu öğrenmesi ve çocuğun Down sendromu için pozitif test yapması durumunda, "Kadın için doğru hareketin hamileliği sonlandırmak olduğunu" tavsiye etti.[11] "Down sendromlu fetüsleri" terimine "sapkın" olarak adlandırmaya karar veren insanlara atıfta bulundu ve "Down sendromlu insan sayısını [artırarak]" "erdemli hiçbir şey" olmadığını söyledi. [12] Ayrıca, "bir fetüste Down sendromu olduğunda, onu durdurup tekrar denemelisiniz" dedi.[13] En az bir kez RMS, Down sendromlu bir çocuğa sahip olmayı evcil hayvan sahibi olmaya benzetti.[14] (Not: 14 numaralı link 25 Mart 2021' de eklendi - 11:33 EDT.)
+
+[11]: https://web.archive.org/web/20210325014348/https://stallman.org/archives/2016-jul-oct.html#31_October_2016_(Down's_syndrome)
+[12]: https://web.archive.org/web/20210325014343/https://stallman.org/archives/2015-jul-oct.html#21_October_2015_(Mistaking_a_fetus_for_a_baby)
+[13]: https://web.archive.org/web/20210325014628/https://stallman.org/archives/2016-mar-jun.html#23_April_2016_(Fetuses_with_Downs_syndrome)
+[14]: https://web.archive.org/web/20161107050933/https://stallman.org/archives/2016-jul-oct.html#31_October_2016_(Down's_syndrome)
+
+RMS, insanların doğru zamirlerini kullanmaya karşı bir kampanya için yıllarını harcadı. Bu, açık bir şekilde transfobidir. GNU Kind Communication Guidelines'ın orijinal yayınında, “İngilizcede üçüncü tekil şahıs zamirlerinde cinsiyet tarafsızlığını ifade etmenin çeşitli yolları vardır; 'onlar'ı (they) kullanmak zorunda değilsiniz." dedi.[15] Bu metin güncellendi, ancak hala transfobiktir.[16] Web sitesindeki ana sayfada " 'Onlar'(they) çoğuldur - tekil öncüller için, cinsiyet ayrımı gözetmeyen tekil zamirler kullanın." İfadesi yer alıyor.[17]
+
+[15]: https://web.archive.org/web/20181022140126/https://www.gnu.org/philosophy/kind-communication.html
+[16]: https://web.archive.org/web/20210325014959/https://www.gnu.org/philosophy/kind-communication.html
+[17]: https://web.archive.org/web/20210325014851/https://stallman.org/
+
+[Ana sayfaya dön][18]
```

Comment:
This seems to be missing quite a bit of text

Reply:
I don't understand exactly which part you mentioned.  I checked  it again and couldn't find any missing part. Can you be more clear. Thanks for your effort.

Reply:
Apologies! It seems to be a copy paste error on my part!